ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DynamicPartitionPruner.java

```diff
@@ -481,7 +482,9 @@ protected String processPayload(ByteBuffer payload, String sourceName) throws Se
             LOG.debug("Adding: {} to list of required partitions", value);
 
             info.values.add(value);
+            partitionCount++;
           }
+          LOG.info("Received {} partitions (source: {})", partitionCount, sourceName);
```

Comment:
LGTM. +1.

Minor comment before commit: It would be nice to include columnName in this line. (info.columnName).